README.md

```diff
@@ -1,23 +1,24 @@
 # ffmpeg-build-script
 
-This build script, is to support the NRCHKB project.  
-in allowing a fully updatable ffmpeg build for multiple pi versions (arm), OS's and intel hosts.
+This build script, is to support the NRCHKB project in allowing a fully updatable FFmpeg builds. 
+It is designed for multiple Raspberry Pi versions (arm), OS's and intel hosts.
 
 Highlights
 
- - FFmpeg 5.0+ (Built for your platform)
- - 32 and 64 bit support
- - libfdk-aac (Built for your platform)
- - libx264 (Installed via apt, or will build from source as a fallback if not available)
- - h264_omx (32Bit only, Raspberry Pi only, deprecated)
- - h264_v4l2m2m (replaces h264_omx)
- - Ability to add extra compile flags
+- FFmpeg 5.0+ (Built for your platform)
+- 32 and 64 bit support
+- libfdk-aac (Built for your platform)
+- libx264 (Installed via apt, or will build from a source as a fallback if not available)
```

Comment:
"Built from **a** source" or "built from source" 😉 

Its "built from source" as opposed to "built from a source"

Reply:
You native, I am not :)
Which form do you prefer? 
I believe you can apply changes.

Reply:
I will add tested systems to the read me, so will make the changes here 👍